examples/actor_critic.py

```diff
@@ -401,11 +383,6 @@ def main(argv):
         print(f"   Avg performance: {statistics.mean(performances):.2f}")
         print(f" Worst performance: {min(performances):.2f}")
 
-    finally:
-        # The program will not terminate until the environment is
-        # closed.
-        env.close()
```

Comment:
Oh nice, I take it this behavior of env is no longer the case? (how?)

Reply:
It is still the case, but I found that gym environments have support for "with statement" usage which automatically closes the environment, so

```py
with gym.make("...") as env:
    do_stuff(env)
```

is equivalent to:

```py
try:
    env = gym.make("...")
    do_stuff(env)
finally:
    env.close()
```